src/screens/FeedScreen/Footer.js

```diff
@@ -12,7 +12,7 @@ import MemoIcShare from '../../assets/icons/Ic_share';
 import MemoIcComment from '../../assets/icons/Ic_comment';
 import MemoIcArrowDownVoteOn from '../../assets/arrow/Ic_downvote_on';
 import MemoIcArrowUpvoteOn from '../../assets/arrow/Ic_upvote_on';
-import {colors} from '../../utils/colors';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to use `COLORS` from the `utils/theme` module instead of `colors`. This change seems to be part of a larger refactoring effort to standardize color usage across the application. As long as `COLORS` is correctly defined and exported in the `utils/theme` module, this change should not introduce any issues.
```diff
- import {colors} from '../../utils/colors';
+ import {COLORS} from '../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->